k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/k4EDM4hep2LcioConv.ipp

```diff
@@ -23,7 +25,15 @@ std::unique_ptr<lcio::LCCollectionVec> convertTracks(const edm4hep::TrackCollect
       lcio_tr->setNdf(edm_tr.getNdf());
       lcio_tr->setdEdx(edm_tr.getDEdx());
       lcio_tr->setdEdxError(edm_tr.getDEdxError());
-      lcio_tr->setRadiusOfInnermostHit(edm_tr.getRadiusOfInnermostHit());
+      double radius = std::numeric_limits<double>::max();
+      for (const auto& hit : edm_tr.getTrackerHits()) {
+        radius = std::min(radius, std::sqrt(hit.getPosition()[0] * hit.getPosition()[0] +
+                                            hit.getPosition()[1] * hit.getPosition()[1]));
+      }
+      if (radius == std::numeric_limits<double>::max()) {
+        radius = 0;
+      }
```

Comment:
Since this is also used in the tests, this could also be some utility function. Potentially even in EDM4hep, where we could directly add it in https://github.com/key4hep/EDM4hep/pull/326.